rest/model/dns/zone.go

```diff
@@ -37,6 +37,9 @@ type Zone struct {
 	Primary *ZonePrimary `json:"primary,omitempty"`
 	// Secondary contains info for slaving the zone to a primary dns server.
 	Secondary *ZoneSecondary `json:"secondary,omitempty"`
+
+	// Whether or not DNSSEC is enabled on the zone
+	DNSSEC bool `json:"dnssec"`
```

Comment:
couldn't use `omitempty` on a bool - prevents us from setting it false. doesn't seem like a back-compatibility issue tho